mobile/lib/modules/album/providers/album_sort_by_options.provider.dart

```diff
@@ -0,0 +1,135 @@
+import 'package:collection/collection.dart';
+import 'package:immich_mobile/modules/settings/providers/app_settings.provider.dart';
+import 'package:immich_mobile/modules/settings/services/app_settings.service.dart';
+import 'package:immich_mobile/shared/models/album.dart';
+import 'package:riverpod_annotation/riverpod_annotation.dart';
+
+part 'album_sort_by_options.provider.g.dart';
+
+typedef AlbumSortFn = List<Album> Function(List<Album> albums, bool isReverse);
+
+class _AlbumSortHandlers {
+  const _AlbumSortHandlers._();
+
+  static const AlbumSortFn created = _sortByCreated;
+  static List<Album> _sortByCreated(List<Album> albums, bool isReverse) {
+    final sorted =
+        albums.where((a) => a.isRemote).sortedBy((album) => album.createdAt);
```

Comment:
where(..is remote) is used everywhere in this file - can we simply leave it out?
Device albums could also be sorted with this.

Reply:
Filtering device out means they won't show at all, so no need to apply any of these functions on them.
If the sorting functions are only to be used on remote albums, we don't need the where(is remote) part.

Reply:
Ah, yes. It should be made that way and the sorting functions should not be filtering the albums passed to it. Will update it. Thank you!

Reply:
Updated!